components/dataflow/base/build_image.sh

```diff
@@ -15,12 +15,12 @@
 
 
 mkdir -p ./build
-rsync -arvp "../../tfdv"/ ./build/
-rsync -arvp "../../tft"/ ./build/
-rsync -arvp "../../tfma"/ ./build/
-rsync -arvp "../../predict"/ ./build/
-cp ../../../license.sh ./build
-cp ../../../third_party_licenses.csv ./build
+rsync -arvp "../tfdv/src"/ ./build/
```

Comment:
In future we should probably try to separate the component images. For now it's fine.

Reply:
are you talking about separating the component directory from the pipeline repo?

Reply:
No. I'm talking about having one image per dataflow component. Right now they're using a single image. But this is low-priority.

Reply:
We do have one image per dataflow component. This is the base image that captures the dependencies. The source code is built into leaf images.